traffic_portal/app/src/styles/main.scss

```diff
@@ -183,6 +183,13 @@ table {
   margin-right: 5px;
 }
 
+input[type="checkbox"].dirty {
+  -moz-box-shadow:    inset 0 0 10px red;
+  -webkit-box-shadow: inset 0 0 10px red;
+  box-shadow:         inset 0 0 10px red;
```

Comment:
Red to me means "invalid"/"wrong" - I think a light blue glow would be better. You could set the "dirty" class on the entire row to make it glow blue, if you're concerned about a clash with host OS theming colors.

Reply:
yeah, understood. i'll play around with colors more.

Reply:
changed to green